### PR TITLE
WIP first draft for json import, needed for wallabag v1 migration

### PR DESCRIPTION
**Pull request by nicosomb**
It already works \o/ via app/console import:json

### CODE REVIEW

File: src/Wallabag/CoreBundle/Command/ImportCommand.php [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+        // Define the size of record, the frequency for persisting the data and the current index of records
+        $size = count($data);
+        $batchSize = 20;
+        $i = 1;
+
+        $progress = new ProgressBar($output, $size);
+        $progress->start();
+
+        $user = $em->getRepository('WallabagCoreBundle:User')
+            ->findOneById(1);
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Command/ImportCommand.php]: :trollface:

Comment (nicosomb) [inline src/Wallabag/CoreBundle/Command/ImportCommand.php]: ;-) did you read my comment on the PR?  :-P

Comment (j0k3r) [inline src/Wallabag/CoreBundle/Command/ImportCommand.php]: I do :)

**Comment by nicosomb** [comment]
Done: 
* form to upload file
* configuration to define which files we can import. 

Todo: 
- [ ] tests
- [x] today, all imported articles are affected to User with 1. Need to be fixed. 
- [x] a link in configuration screen to upload the file and launch the command
- [ ] delete the import file after the import
- [ ] move all the logic into new ImportBundle

This PR will be linked to #1493.

**Comment by nicosomb** [comment]
Started documentation for this feature. http://wallabag.readthedocs.org/en/v2-import-from-json/user/import.html

**Comment by spitfire** [comment]
Hi. I've checked out v2-import-from-json and tried it on a ~9MB json file (adjusted upload size in php.ini) using web interface and got "Error while processing import". How do I collect (hopefully useful) information?

How can I use the console import feature?

**Comment by j0k3r** [comment]
@nicosomb I think we should close this PR and report your work on #1493 so everything related to import will be handle in that PR. What do you think?

**Comment by nicosomb** [comment]
I think you're right, as usual.

**Pull request closed without merging**